ci-operator/step-registry/dotnet/execute-tests/dotnet-execute-tests-commands.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
```

Comment:
If you add the sleep here then you can manually run these two commands in the pods terminal and debug any failures/try different things in realtime